### ISSUE TITLE
TailwindCSS classes gives error on build

### ISSUE THREAD

**Issue by AdrianMrn** [laravel-filament/filament#1513]
### Package

filament/forms

### Package Version

v2.10.0

### Laravel Version

v8.82.0

### PHP Version

v2.10.1

### Bug description

Since this commit https://github.com/laravel-filament/filament/commit/952dbc8e36c055ea8a18f99f879dadf7c6d2f776 (I think) where TailwindCSS classes were added, our build fails because we've configured a custom theme in our Tailwind config, which doesn't have the color `border-gray-300` (and the other colors that are used in the css).

This would also break for people who don't use TailwindCSS or postcss-import in their project (because of the `@apply` statements).

Would it be possible to revert back to plain css classes? 

### Steps to reproduce

Import the css file and build it while using a custom tailwindcss config that overrides colors.

### Relevant log output

```shell
The `border-gray-300` class does not exist. If `border-gray-300` is a custom class, make sure it is defined within a `@layer` directive.
```

**Comment by ryangjchandler** [comment]
@AdrianMrn I don't think we can offer full BC support for users who have built their own CSS. We don't control your Tailwind config file and therefore can't control your build output.

Based on your problem, we'd never be able to introduce/use a new Tailwind class. We assume users are following the TW instructions from our docs, which include everything that you need to for Tailwind to work with Filament.

**Comment by danharrin** [comment]
Yeah I agree with Ryan - I'm a bit confused about how you're using Filament _at all_ without a gray class - we use it across pretty much every view? Have you overriden every single one of those classes? 😬

**Comment by AdrianMrn** [comment]
![homer-simpson-bush-gif](https://user-images.githubusercontent.com/12762044/153169869-fd8ad245-5432-418f-92fb-e6ddd0633a43.gif)

Thanks for the quick replies, I now realise we'd be running into styling issues regardless of build errors :)
I'll try to find a way to fix this on our end, you're right that this is not the package's problem at all.

We've got a pretty complex app that a lot of different people have been working on. The reason the Tailwind color classes are overridden is so we can only use the color classes we created ourselves, so they can be changed through some config values, because the app will be deployed in a multitenancy context. I'll see if we can instead extend the default Tailwindcss theme so we don't remove the default classes 👍

Again thanks for the super quick replies!
